lua/telescope/builtin/files.lua

```diff
@@ -53,6 +53,26 @@ files.grep_string = function(opts)
   }):find()
 end
 
+-- NOTE(tj): Get clarification on comment and put this in the right place
+local function insert_search_dirs(find_command, search_dirs)
+  local command = find_command[1]
+  local insert_pos = -1
+  if command == 'fd' or command == 'fdfind' then
+    table.insert(find_command, '.')
```

Comment:
I think we should not have a separate function like this. but in the `find_files` section, we can change what the `find_command` is directly.

So on line 82 right now, you can do:

```lua
if search_dirs then
  find_command = { 'fd, '--type', 'f', '.' }
else
  find_command = {'fd', '--type', 'f'}
end
```
or similar.

Does that make sense? I don't like how this function lives out here but has exactly the same `if/elseif/else` idea as in `find_files`.

Reply:
Okay got it

Reply:
Hmmm. Doing it hard-coded that way makes it impossible to use `search_dirs` for custom commands. Or did you mean it figuratively, as in, should I just replicate the logic I have used in the function, just in the main `if` block in `find_files`? It might lead to some repeated code

Reply:
if you pass a custom command, you should make it work with search_dirs. I think that is the fair way to implement. I would not worry about custom commands.

Reply:
(by you, I mean, the user, in the above comment).

I prefer the hard coded way.